library/src/main/java/com/bumptech/glide/RequestBuilder.java

```diff
@@ -571,8 +574,9 @@ protected RequestOptions getMutableOptions() {
     return into(target, /*targetListener=*/ null);
   }
 
+  @RestrictTo(Scope.LIBRARY)
```

Comment:
I'm ok not using this for non-public methods. I assume anything that's non-public can be broken and even some of the public APIs we have are breakable (those really should be marked with ``@RestrictTo`` or some equivalent.

Fine to keep it, just as a note for the future (or a discussion point).

Reply:
Lot of negation there :)
You're right: `default` methods don't need this, if someone uses "internal" (name/keyword for default visibility in other languages) API, they can fix it if we break something. `protected` on public classes may need this, and `public` methods that are only public because of packages, and not because it's public API also need this. I'll remove from `default` ones.